lib/tasks/after_tasks.rake

```diff
@@ -874,6 +874,34 @@ namespace :After do
     puts("Added default rating to works: #{updated_works}") && STDOUT.flush
   end
 
+  desc "Delete invalid icon data from pseuds"
+  task(delete_invalid_pseud_icon_data: :environment) do
+    # From validates_attachment_content_type in pseuds model.
+    valid_types = %w[image/gif image/jpeg image/png]
```

Comment:
We have a tiny few entries with "image/jpg", so we should edit them to "image/jpeg".